ui/web-v2/apps/admin/src/components/FeatureCloneForm/index.tsx

```diff
@@ -89,10 +89,10 @@ export const FeatureCloneForm: FC<FeatureCloneFormProps> = memo(
                   <div className="mt-1">
                     <input
                       type="text"
-                      name="originEnvironmentId"
-                      id="originEnvironmentId"
+                      name="originEnvironmentName"
+                      id="originEnvironmentName"
```

Comment:
The `originEnvironmentId` field is not used for executing clone API.
https://github.com/bucketeer-io/bucketeer/blob/3b5fb97d212c5cae7e3488d47e9cd1897c17c296/ui/web-v2/apps/admin/src/pages/feature/index.tsx#L347-L371